### PR TITLE
site setting menu test

### PR DESCRIPTION
**Pull request by nkrulikovskiy**
- Test of tokenclaim button on top menu
- Test of site setting menu on the top menu.

### CODE REVIEW

File: src/components/TopMenu.tsx [hunk 1]
```diff
@@ -128,6 +133,7 @@ function IconButtonAndSettings(): ReactElement {
   return (
     <div style={{ position: "relative" }} ref={wrapperRef}>
       <Button
+        data-testid="settingMenuBtn"
```
Comment (ug02fast) [inline src/components/TopMenu.tsx:136]: `settingsMenuBtn` for consistency.

File: cypress/integration/topMenu.test.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+context("Tob menu test", () => {
```
[the remark this answers is not in the document]
Comment (nkrulikovskiy) [reply cypress/integration/topMenu.test.ts:1]: Oh, my typo

File: cypress/integration/topMenu.test.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  beforeEach(() => {
+    const host = Cypress.env("DAPP_HOST") as string
+
+    cy.visit(`${host}#/pools`)
+    cy.wait(3000)
+  })
+  it("render top menu ", () => {
```
[the remark this answers is not in the document]
Comment (nkrulikovskiy) [reply cypress/integration/topMenu.test.ts:8]: Good habit!

File: cypress/integration/topMenu.test.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+    cy.get("[data-testid=tknClaimContainer]")
+      .should("exist")
+      .find("[data-testid=tokenAddBtn]")
+      .click()
+    cy.get("[data-testid=modalContainer]")
+      .should("be.visible")
+      .find("[data-testid=modalCloseBtn]")
+      .click()
+      .should("not.exist")
```
[the remark this answers is not in the document]
Comment (nkrulikovskiy) [reply cypress/integration/topMenu.test.ts:23]: It is testing modal close when click close button.

**Review by ug02fast — APPROVED** [review]

**Review by hammeiam — APPROVED** [review]

**Pull request merged**